### PR TITLE
chore: add redirect for swap, exchange & gas

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-8028

### CODE REVIEW

File: src/middleware.ts [hunk 1]
```diff
@@ -2,18 +2,32 @@
...
+  const urlPath = request.nextUrl.pathname;
 
+  // Check if the path is either "/exchange" or "/swap"
+  if (urlPath.includes('/exchange/') || urlPath.includes('/swap/')) {
+    // Redirect to the root path "/"
+    return NextResponse.redirect(new URL('/', request.url));
+  }
+  // Check if the path is either "/gas"
+  if (urlPath.includes('/gas/')) {
+    // Redirect to the default gas-tab "/refuel"
```
Comment (tcheee) [inline src/middleware.ts:22]: let's keep "/gas" rather than "/refuel" @dennyscode

**Comment by tcheee** [comment]
We should also rename the "refuel" folder in "gas" in this PR

File: src/middleware.ts [hunk 2]
```diff
@@ -2,18 +2,32 @@
...
 
 export function middleware(request: NextRequest) {
-  request.headers.set('x-current-url', request.url);
+  // Extract the URL path
+  const urlPath = request.nextUrl.pathname;
 
+  // Check if the path is either "/exchange" or "/swap"
+  if (urlPath.includes('/exchange/') || urlPath.includes('/swap/')) {
+    // Redirect to the root path "/"
+    return NextResponse.redirect(new URL('/', request.url));
```
[the remark this answers is not in the document]
Comment (dennyscode) [reply src/middleware.ts]: Nice! Thanks for that

**Review by tcheee — APPROVED** [review]

File: next.config.mjs [hunk 3]
```diff
@@ -47,6 +47,40 @@ const nextConfig = {
...
+        source: '/:lng/exchange',
+        destination: '/',
+        permanent: true,
+      },
+      {
+        source: '/:lng/refuel',
+        destination: '/gas',
+        permanent: true,
+      },
+    ];
```
Comment (oktapodia) [inline next.config.mjs:82]:
```suggestion
    return [
      {
        source: '/:lng?/swap',
        destination: '/',
        permanent: true,
      },
      {
        source: '/:lng?/exchange',
        destination: '/',
        permanent: true,
      },
      {
        source: '/:lng?/refuel',
        destination: '/gas',
        permanent: true,
      },
    ];
```

**Review by oktapodia — CHANGES_REQUESTED** [review]
Just a quick comment

**Review by oktapodia — APPROVED** [review]
LGTM

**Pull request merged**